generator/internal/genclient/templatedata.go

```diff
@@ -145,6 +149,10 @@ func (s *service) NameToCamel() string {
 	return s.c.ToCamel(s.s.Name)
 }
 
+func (s *service) Name() string {
+	return s.s.Name
+}
```

Comment:
Should this go through the Codec to keep the right style?  All languages use `PascalCase` for these (and Protobuf too), so maybe we can postpone that change until we run into a language that insists on `snake_case` for classes too.  Okay, I answered my own question, leave as-is.

Reply:
changed to service name to read better in template.